src/dynamic-plugin-sdk.ts

```diff
@@ -609,7 +596,7 @@ export const k8sListResource = adapterFunc(k8sList, ['model', 'queryParams', 'ra
 export const useK8sWatchResource = <R extends K8sResourceCommon | K8sResourceCommon[]>(initResource: WatchK8sResource): WatchK8sResult<R> => {
   const resource = useDeepCompareMemoize(initResource, true);
   const resourceRef = React.useRef<R | null>(null);
-  const fetchedRef = React.useRef<boolean>(false);
+  const [fetched, setFetched] = React.useState<boolean>(false);
```

Comment:
reason for changing it from ref to state?

Reply:
Switched to useState since ref does not seem to trigger rerenders when data is fetched, and loaded was always set to false.